Sources/TuistKit/Services/Cache/CacheWarmService.swift

```diff
@@ -41,7 +43,50 @@ final class CacheWarmService {
         }
     }
 
+    private func cacheFlavor(named flavorName: String?, from config: Config) throws -> TuistCore.Cache.Flavor {
+        let flavors = config.cache.flavors
+        switch flavorName {
+        case .none:
+            guard let defaultFlavor = flavors.first else {
+                throw CacheWarmServiceError.missingDefaultFlavor
```

Comment:
Remember to document this on the documentation website. This is a convention that we are defining and that is important to document.

Reply:
Can you please show me an example on the documentation website? 🤔 Not sure how to document it.